OData/src/System.Web.OData/OData/Batch/OperationResponseItem.cs

```diff
@@ -48,6 +48,14 @@ public override Task WriteResponseAsync(ODataBatchWriter writer, CancellationTok
             return WriteMessageAsync(writer, Response, cancellationToken);
         }
 
+        /// <summary>
+        /// Gets a value that indicates if the HTTP response in response item was successful.
+        /// </summary>
+        internal override bool IsResponseSuccessful()
```

Comment:
@xuzhg It's an override method that overrides the base member, I did this in order to keep consistency of the interface.